minethd.cpp

```diff
@@ -530,3 +560,446 @@ void minethd::double_work_main()
 	cryptonight_free_ctx(ctx0);
 	cryptonight_free_ctx(ctx1);
 }
+
```

Comment:
Where are this changes coming from?
Is this part of the fix or is it a mistake that this is addded here.